{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "2ea99a3c_b2202738",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 7
      },
      "lineNbr": 0,
      "author": {
        "id": 5290
      },
      "writtenOn": "2020-12-03T18:34:46Z",
      "side": 1,
      "message": "Pls add \"BUG\u003daomedia:2886\" to the commit message, as this patch will fix warning (5) mentioned there I believe.",
      "revId": "beeb4e4fcc42695f96f995b661cdacf8502cdc86",
      "serverId": "e5514cf8-2d6e-3e29-adb4-24cd6dde4bf0"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "6e9a7d8d_2779ac46",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 7
      },
      "lineNbr": 0,
      "author": {
        "id": 21727
      },
      "writtenOn": "2020-12-04T09:42:16Z",
      "side": 1,
      "message": "Thanks for your suggestion. I will add it in commit message.",
      "revId": "beeb4e4fcc42695f96f995b661cdacf8502cdc86",
      "serverId": "e5514cf8-2d6e-3e29-adb4-24cd6dde4bf0"
    }
  ]
}